test/Microsoft.Azure.SignalR.AspNet.Tests/Infrastructure/TestServiceMessageHandler.cs

```diff
@@ -46,7 +46,7 @@ public override Task WriteAsync(string partitionKey, ServiceMessage serviceMessa
                 throw new InvalidOperationException("Not expected to write before tcs is inited");
             }
 
-            return Task.CompletedTask;
+            return Task.FromResult(true);
```

Comment:
This allocates. Change Task<bool> to ValueTask<bool>

Reply:
This just in test. In other situations, we need to allocate a task.